app/components/modal/edit-user/component.js

```diff
@@ -0,0 +1,39 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { userType } from 'mms-ember-project/utils/form-data';
+
+export default class ModalEditUserComponent extends Component {
+  userType = userType;
```

Comment:
Nice idea with this import! ❤️ One thing:
```suggestion
import { userTypes } from 'mms-ember-project/utils/form-data';

export default class ModalEditUserComponent extends Component {
  userTypes = userTypes;
```